Cargo.toml

```diff
@@ -12,3 +12,4 @@ pnet = "0.35.0"
 log = "0.4.22"
 thiserror = "1.0.61"
 anyhow = "1.0.86"
+colog = "1.3"
```

Comment:
Added `colog`, but it's easy to use any other [provider](https://docs.rs/log/latest/log/#available-logging-implementations), doesn't require much changes mainly initialization